electron-src/helpers/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const ipcMainChannels = {
```

Comment:
Could you please make the names of constants in upper case? 
Examples:
IPC_MAIN_CHANNELS = {}
REDIRECT: 'redirect'
GET_CURRENT_VERSION: get'CurrentVersion'